src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -240,14 +242,24 @@ export function validateAddress(errors, formData) {
 export const download4142Notice = (
   <div className="usa-alert usa-alert-warning no-background-image">
     <p>
-      Since your doctor has your private medical records, you’ll need to fill
+      Since your doctor has your private medical records, you'll need to fill
       out an Authorization to Disclose Information to the VA (VA Form 21-4142) so
-      we can request your records. You’ll need to fill out a form for each doctor.
```

Comment:
I thought we wanted to include the part about filling out one form per doctor? Just checking.

Reply:
Ohhh, I added it back to the wrong place. Good catch.